src/components/channel/ChanelCreate.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { useNavigate, useParams } from "react-router-dom";
+import ChannelForm from "./ChannelForm";
```

Comment:
```suggestion
import { useNavigate, useParams } from "react-router-dom";
import PropTypes from "prop-types";

import ChannelForm from "./ChannelForm";
```
저희 컨벤션에 맞게 라이브러리 import 이후 한 줄 개행해주시면 어떨까해요~ 동일 내용은 생략하겠습니다!

Reply:
예리하시네요 좋아요~